koku/api/resource_types/test/tests_views.py

```diff
@@ -24,16 +24,16 @@
 
 
 class ResourceTypesViewTest(IamTestCase):
-    """Tests the resource types view."""
+    """Tests the report view."""
 
     ENDPOINTS_RTYPE = ["resource-types"]
-    ENDPOINTS_AWS = ["aws-accounts", "aws-organizational-units"]
+    ENDPOINTS_AWS = ["aws-accounts"]  # , "aws-organizational-units"]
```

Comment:
Is there a reason to leave in the commented-out part of the change? 
If so, please leave an extra comment to that effect.

Reply:
I had left it commented out since it would be the next api that I worked on. I can remove it for now and add it back when this API is added!